jetty-start/src/main/java/org/eclipse/jetty/start/Main.java

```diff
@@ -467,6 +468,17 @@ else if (args.isCreateFiles() || !args.getStartModules().isEmpty())
         {
             CommandLineBuilder cmd = args.getMainArgs(StartArgs.ALL_PARTS);
             cmd.debug();
+
+            List<String> execModules = args.getEnabledModules().stream()
+                .map(name -> args.getAllModules().get(name))
+                // Keep only the forking modules.
+                .filter(module -> !module.getJvmArgs().isEmpty())
+                .map(Module::getName)
+                .collect(Collectors.toList());
+            StartLog.warn("Forking second JVM due to forking module(s): %s.", execModules);
+            StartLog.warn("Consider using option --dry-run to generate the command line:");
+            StartLog.warn("$(java -jar $JETTY_HOME/start.jar --dry-run)");
```

Comment:
3 lines of warning is a bit extreme!
```suggestion
            StartLog.warn("Forking JVM due to exec in module(s): %s.  Use --dry-run to generate non forking command line.", execModules);
```